### PR TITLE
Enabling Portuguese in DateTime recognizers.

### PR DESCRIPTION
**Pull request by tellarin**
Enabling PT in timex & adding try catches.
Issue #11.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs [hunk 1]
```diff
@@ -80,24 +87,35 @@ private DateTimeRecognizer(string cultureCode, DateTimeOptions options)
...
+
                 case Culture.French:
                     RegisterModel(Culture.French, type, options.ToString(), new DateTimeModel(
                                       new BaseMergedParser(new FrenchMergedParserConfiguration(options)),
                                       new BaseMergedExtractor(new FrenchMergedExtractorConfiguration(options))
                                   ));
                     break;
+
+                case Culture.Portuguese:
+                    RegisterModel(Culture.French, type, options.ToString(), new DateTimeModel(
```
Comment (yiylin) [inline .NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs]: French？

File: Specs/DateTime/Portuguese/DateExtractor.json [hunk 2]
```diff
@@ -34,6 +34,20 @@
   ]
 }
 ,
+{
+  "Input": "nas ultimas 3 semanas",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [  ]
```
Comment (yiylin) [inline Specs/DateTime/Portuguese/DateExtractor.json]: Cause exception, so no result?

Comment (tellarin) [reply Specs/DateTime/Portuguese/DateExtractor.json]: Nope. This should never be extracted as this type.

File: .NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs (hunk 1 above)
Comment (tellarin) [reply .NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs]: Fixed.

**Pull request merged**